src/app/lib/config/services/polska.ts

```diff
@@ -384,6 +384,12 @@ export const service: DefaultServiceConfig = {
         'BBC. BBC nie ponosi odpowiedzialności za treści na stronach zewnętrznych.',
     },
     timezone: 'Europe/Warsaw',
+    navigation: [
+      {
+        title: 'Strona główna',
+        url: '/polska',
+      },
+    ],
```

Comment:
until go live on tues 24th , this url will keep redirecting to the about page like this - bbc.com/polska , we made a decision to remove the navigation until the full site is live as it feels better from a ux perspective, we have a ticket to reinstate this nav before live though - so if we could remove this, and i will make sure it's added back when we do that!

Reply:
good point ! here's the [ticket](https://bbc.atlassian.net/browse/WS-411)  , and a [link to the discussion](https://github.com/bbc/simorgh/pull/12796#discussion_r2099988761) we had about removing this

Reply:
I see, ok, let me remove the nav